coverage.mixin

```diff
@@ -6,5 +6,10 @@
                 "-DCMAKE_CXX_FLAGS='--coverage'"
             ]
         }
+    },
+    "test": {
+        "coverage-py": {
```

Comment:
As long as this mixin only supports enabling coverage for `pytest` maybe rename it to `coverage-pytest`?

Reply:
My thinking was to make the name generic to allow it to be extended in the future.

Renamed in https://github.com/colcon/colcon-mixin-repository/pull/21/commits/2034da0eadb7c894f7586b88d87baf187b22cda0

Reply:
Let's add the generic variant when it becomes generic :wink: